cdap-docs/admin-manual/source/installation/installation-note.txt

```diff
@@ -0,0 +1,21 @@
+.. topic:: Different ways to install CDAP
```

Comment:
As it is txt, Sphinx will ignore it, and not complain about an RST file that is an orphan or not included in a table of contents. It won't try to render it by itself. When I have a large number of these (such as in the examples) I put them in a separate directory. But by putting it as txt, other programs know what I would like done with it, when editing etc.